packages/tailwindcss-language-server/src/resolve-css-imports.ts

```diff
@@ -15,8 +18,129 @@ const resolveImports = postcss([
       return paths ? paths : id
     },
   }),
+  fixRelativePathsPlugin(),
 ])
 
 export function resolveCssImports() {
   return resolveImports
 }
+
+const SINGLE_QUOTE = "'"
+const DOUBLE_QUOTE = '"'
+
+export default function fixRelativePathsPlugin(): Plugin {
```

Comment:
This should live in a separate file. Probably make a `src/css` folder or something and put resolve-css-imports.ts in there as well